controllers/ebpf/internal/permissions/permissions.go

```diff
@@ -155,12 +155,10 @@ func (c *Reconciler) reconcileOpenshiftPermissions(
 	} else {
 		scc.AllowedCapabilities = AllowedCapabilities
 	}
-	if (desired.EnablePktDrop != nil && *desired.EnablePktDrop) ||
-		(desired.EnableDNSTracking != nil && *desired.EnableDNSTracking) {
+	if helper.IsFeatureEnabled(desired.PktDrop) {
 		scc.AllowHostDirVolumePlugin = true
 	}
-	if (desired.EnablePktDrop != nil && !*desired.EnablePktDrop) &&
-		(desired.EnableDNSTracking != nil && !*desired.EnableDNSTracking) {
+	if helper.IsFeatureEnabled(desired.DNSTracking) {
 		scc.AllowHostDirVolumePlugin = false
 	}
```

Comment:
Hmm that doesn't look good .. we want AllowHostDirVolumePlugin for any of PktDrop or DNS
And actually the second `if` looks useless, I think

Reply:
I had missed this. Thanks for you commit I have merged that one in this branch.